_posts/2020-02-10-openssh-8.2.md

```diff
@@ -0,0 +1,16 @@
+---
+title: OpenSSH 8.2 update requires restart of sshd
+layout: post
+---
+
+With the OpenSSH update to 8.2 restarting the `sshd`
+service is required to accept new ssh connections.
```

Comment:
Probably reword as follows for more impact:

```
With the OpenSSH update to 8.2 you *must* restart the
`sshd` service, or it will not accept new ssh connections.
```